clients.json

```
@@ -1875,5 +1875,15 @@
     "description": "A custom connector for Dell Boomi that utilizes the lettuce.io Java client to add Redis client support to the Dell Boomi iPaaS.",
     "authors": [],
     "active": true
+  },
+  
+  {
+    "name": "FreeRedis",
+    "language": "C#",
+    "url": "https://github.com/2881099/FreeRedis",
+    "description": "This .NET client supports redis6.0+, cluster, sentinel, pipeline, And simple api.",
+    "authors": ["2881099"],
```

This should contain one or more Twitter handles. If you don't use it, just leave it empty likeso:
```suggestion
    "authors": [],
```